ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -235,7 +236,8 @@ class WalletHomeScreen extends React.Component<Props, never> {
 
 const mapStateToProps = (state: GlobalState) => ({
   potWallets: walletsSelector(state),
-  potTransactions: latestTransactionsSelector(state)
+  potTransactions: latestTransactionsSelector(state),
+  isPagoPATestEnabled: state.persistedPreferences.isPagoPATestEnabled
```

Comment:
```suggestion
  isPagoPATestEnabled: isPagoPATestEnabledSelector(state)
```

Reply:
@valentinacala yes this is a good suggestion, please use the selector also in `WalletHomeScreen`